beacon_node/store/src/hot_cold_store.rs

```diff
@@ -2156,19 +2182,17 @@ impl<E: EthSpec, Hot: ItemStore<E>, Cold: ItemStore<E>> HotColdDB<E, Hot, Cold>
                 Err(e) => {
                     warn!(
                         self.log,
-                        "Stopping blobs sidecar pruning early";
+                        "Stopping blob pruning early";
                         "error" => ?e,
                     );
                     break;
                 }
             };
 
-            if Some(block_root) != last_pruned_block_root
-                && self.blobs_sidecar_exists(&block_root)?
-            {
+            if Some(block_root) != last_pruned_block_root && self.blobs_exist(&block_root)? {
                 debug!(
```

Comment:
should this be a trace if we're emitting it up to once per block on a prune? Could be pretty spammy